packages/dds/merge-tree/src/mergeTreeDeltaCallback.ts

```diff
@@ -25,6 +25,7 @@ export const enum MergeTreeMaintenanceType {
      *    b) The segment's tracking collection is empty (e.g., not being tracked for undo/redo).
      */
     UNLINK  = -3,
+    ACKNOWLEDGED = -4,
```

Comment:
Naming is hard. I'm not sure if "acknowledged" is something that consumers can relate to?
At minimum, please put a comment describing when it is called

Reply:
Added comment